### PR TITLE
fedekunze/ 2000 distribution endpoints

### LINKED ISSUE #2000: Add distribution endpoints for rewards
Description:

Add to `lcdClient`

[List of endpoints](https://cosmos.network/rpc/#/ICS24)

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #2000

**Description:**

Thank you! 🚀

---

For contributor:

- [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [x] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/connectors/lcdClient.js [hunk 1]
```diff
@@ -204,8 +204,58 @@ const Client = (axios, remoteLcdURL) => {
...
     },
+    /* ============ Explorer ============ */
     getBlock: function(blockHeight) {
       return req(`GET`, `/blocks/${blockHeight}`)()
+    },
+    /* ============ Distribution ============ */
+    getDelegatorRewards: function(delegatorAddr) {
+      return req(`GET`, `/distribution/delegators/${delegatorAddr}/rewards`)()
+    },
+    postDelegatorRewards: function(delegatorAddr, data) {
```
Comment (faboweb) [inline app/src/renderer/connectors/lcdClient.js]: what does this do?

Comment (fedekunze) [reply app/src/renderer/connectors/lcdClient.js]: withdraws all rewards from a delegator

Comment (fedekunze) [reply app/src/renderer/connectors/lcdClient.js]: from all the delegations at once

Comment (faboweb) [reply app/src/renderer/connectors/lcdClient.js]: how about:
```suggestion
    postDelegationRewardsClaim: function(delegatorAddr, data) {
```

Comment (faboweb) [reply app/src/renderer/connectors/lcdClient.js]: or:
```suggestion
    postDelegatorRewardsClaim: function(delegatorAddr, data) {
```

Comment (fedekunze) [reply app/src/renderer/connectors/lcdClient.js]: what about `postWithdrawDelegatorRewards` ?

Comment (faboweb) [reply app/src/renderer/connectors/lcdClient.js]: good for me

Comment (fedekunze) [reply app/src/renderer/connectors/lcdClient.js]: updated

**Pull request merged**